### PR TITLE
Add test unauth user download from unprotected repo

### PR DESCRIPTION
**Pull request by swadeley**
For 6.10 user stories

"Content Protection:
As an unauthenticated user, I can access unprotected content."

Thank you

### CODE REVIEW

File: tests/foreman/api/test_repository.py [hunk 1]
```diff
@@ -1234,6 +1236,46 @@ def test_positive_access_protected_repository(self, module_org, repo, default_sa
...
+    def test_positive_access_unprotected_repository(self, module_org, repo, default_sat):
+        """Access files in unprotected repository over HTTP and HTTPS
+
+        :id: 43fe24c8-7a50-4d38-8259-b23e5ed5800a
+
+        :parametrized: yes
+
+        :expectedresults: The repository data file is successfully accessed.
+
+
```
Comment (pondrejk) [inline tests/foreman/api/test_repository.py:1260]: one line too many?

**Review by mshriver — APPROVED** [review]

File: tests/foreman/api/test_repository.py (hunk 1 above)
Comment (swadeley) [reply tests/foreman/api/test_repository.py]: Ok, fixed that, thanks for the review

**Comment by swadeley** [comment]
trigger: test-robottelo
pytest: tests/foreman/api/test_repository.py -k test_positive_access_unprotected_repository

**Comment by JacobCallahan** [comment]
@pondrejk can you revisit this?

**Review by rplevka — APPROVED** [review]

**Comment by swadeley** [comment]
Thank you

**Pull request merged**